client/src/components/layout/BuildTile.js

```diff
@@ -2,11 +2,12 @@ import React from "react"
 import { Link } from "react-router-dom"
 
 const BuildTile = ({ build }) => {
-
     const { id } = build
+    const { userId } = build
```

Comment:
haha no, you can do this on one line! `const { id, userId } = build`

Reply:
userId variable removed. Not exactly sure what that was for.